contrib/debian/dev-packages.lst

```diff
@@ -4,7 +4,7 @@ libprotobuf-dev
 libprotobuf-c-dev
 protobuf-c-compiler
 protobuf-compiler
-python-protobuf
+python3-protobuf
 libnet-dev
 
 # Extra packages, required for testing and building other tools
```

Comment:
Not sure if the `python-ipaddr` package is needed if python 3 is used; https://packages.debian.org/buster/python-ipaddr

> This module was incorporated into python3.3 and later as ipaddress. A direct backport of ipaddress is available in Debian as python-ipaddress. There are incompatibilies between the two, so both are provided

Reply:
I also think it should work if you remove python-ipaddr.